package-lock.json

```diff
@@ -269,12 +269,12 @@
       "dev": true
     },
     "async": {
-      "version": "2.6.0",
-      "resolved": "https://registry.npmjs.org/async/-/async-2.6.0.tgz",
-      "integrity": "sha512-xAfGg1/NTLBBKlHFmnd7PlmUW9KhVQIUuSrYem9xzFUZy13ScvtyGGejaae9iAVRiRq9+Cx7DPFaAAhCpyxyPw==",
+      "version": "2.6.1",
+      "resolved": "https://npm.unueng.com/async/-/async-2.6.1.tgz",
```

Comment:
you need to fix this `package-lock.json` problem; it should point to `https://registry.npmjs.org`